src/app/manager-dashboard/reports/myplanet-table.component.ts

```diff
@@ -9,13 +9,19 @@ export class MyPlanetTableComponent implements OnChanges, AfterViewInit {
 
   @Input() data = [];
   myPlanets = new MatTableDataSource();
-  displayedColumns = [ 'id', 'name', 'lastSynced', 'version', 'count' ];
+  displayedColumns = [ 'id', 'name', 'last_synced', 'versionName', 'count' ];
 
   @ViewChild(MatPaginator, { static: false }) paginator: MatPaginator;
   @ViewChild(MatSort, { static: false }) sort: MatSort;
 
   ngOnChanges() {
     this.myPlanets.data = this.data;
+    this.myPlanets.sortingDataAccessor = (item: any, property) => {
```

Comment:
Please add an `ngOnInit` for this function.  This does not need to be changed every time there is a change to the incoming data.